example/resources/data/README.md

```diff
@@ -0,0 +1,16 @@
+# File types
+Different file types which are uploaded to stack in current example
```

Comment:
What is stack and current example?

Reply:
Sry copy-paste from presto module. I will fix it